_config.yml

```diff
@@ -6,7 +6,9 @@ canonical_domain: www.login.gov
 # i18n
 languages:
   - en
-  # - es # disabled in production for now
+  - es
+  - fr
```

Comment:
Just for posterity -- this is overridden by settings in Federalist, so this in an of itself is not launching FR and ES in production